drift_dev/lib/src/writer/schema_version_writer.dart

```diff
@@ -314,6 +325,14 @@ class SchemaVersionWriter {
       final versionClass = '_S$versionNo';
       final versionScope = libraryScope.child();
 
+      versionScope.reserveNames([
```

Comment:
Let's add a small comment here explaining that these names are inherited from the schema class, which is why we want to prevent them from being used for schema entities.